packages/multi-column/src/extensions/DropCursor/MultiColumnDropCursorPlugin.ts

```diff
@@ -0,0 +1,454 @@
+import type { BlockNoteEditor } from "@blocknote/core";
+import {
+  UniqueID,
+  getBlockInfo,
+  getNearestBlockContainerPos,
+  nodeToBlock,
+} from "@blocknote/core";
+import { EditorState, Plugin } from "prosemirror-state";
+import { dropPoint } from "prosemirror-transform";
+import { EditorView } from "prosemirror-view";
+
+const PERCENTAGE_OF_BLOCK_WIDTH_CONSIDERED_SIDE_DROP = 0.1;
```

Comment:
TODO: comment that this is a modified version of https://github.com/ProseMirror/prosemirror-dropcursor/blob/2fb36741ec9bad33de370bbb1802e35eacd85257/src/dropcursor.ts

The logic that has been added is to show "vertical" drop targets (i.e.: "create new column") when dragging over the side of a block